### PR TITLE
Initial support for manipulating access objects

### LINKED ISSUE #40: Backend Support for Access Policies
- [ ] Model
- [ ] REST controllers - POST, PUT, GET, DELETE
- [ ] Consistency rules

### PR DESCRIPTION
**Pull request by asgeirn**
Closes #40 

- [x] Update LDAP schema for Client
- [x] Move to fint-portal-api
- [x] Implement bidirectional linking between Access and Client
- [x] Retrieve main classes per component from admin portal
- [x] Depends on FINTLabs/fint-admin-portal-backend#15

### CODE REVIEW

**Comment by asgeirn** [comment]
@OlaStalberg & @fsjovatsen - where should the REST API call to set an access policy for a client be?

Is it:
- `PUT /api/access/{orgName}/{accessId}/client/{clientName}`
or
- `PUT /api/clients/{orgName}/{clientName}/access/{accessId}`
?

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-USER root
-COPY . .
-COPY --from=client /src/build/ src/main/resources/public/
-RUN gradle --no-daemon build
-
-FROM gcr.io/distroless/java:8
-ENV JAVA_TOOL_OPTIONS -XX:+ExitOnOutOfMemoryError
-COPY --from=java /home/gradle/build/libs/fint-kunde-portal-*.jar /data/fint-kunde-portal.jar
-CMD ["/data/fint-kunde-portal.jar"]
+FROM fintlabsacr.azurecr.io/kunde-portal-frontend:PR-48 as client
```
Comment (OlaStalberg) [inline Dockerfile:1]: Is PR-48 correct here if the branch is deleted and merged?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
-            }
-        }
-    }
+pipeline {
+    agent { label 'docker' }
+    stages {
+        stage('Build') {
+            steps {
+                withDockerRegistry([credentialsId: 'fintlabsacr.azurecr.io', url: 'https://fintlabsacr.azurecr.io']) {
+                    sh "docker pull fintlabsacr.azurecr.io/kunde-portal-frontend:PR-48"
```
Comment (OlaStalberg) [inline Jenkinsfile:7]: Is PR-48 correct here if the branch is deleted and merged?

**Comment by asgeirn** [comment]
@OlaStalberg No, we need to set the image tag to `latest` before merging.

**Comment by asgeirn** [comment]
Closes #40

**Review by asgeirn — COMMENTED** [review]
Using #46 we no longer need this trick in `Dockerfile` and `Jenkinsfile`.

**Comment by asgeirn** [comment]
https://fintlabs.atlassian.net/browse/FST-67

File: src/main/java/no/fint/portal/model/ComponentConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package no.fint.portal.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ComponentConfiguration {
```
[the remark this answers is not in the document]
Comment (fsjovatsen) [reply src/main/java/no/fint/portal/model/ComponentConfiguration.java:8]: yepp

**Pull request merged**